### PR TITLE
Add website docs for MOOSE Systems.

### PR DESCRIPTION
**Pull request by leorachapuis**
Document the following MOOSE Systems on the website : 
- AuxKernels
- AuxVariables
- Bfs
- DGKernel
- DiracKernel
- Functions
- ICs
- MeshModifiers
- MultiApps
- Postprocessors

Refs #9638.

### CODE REVIEW

File: framework/doc/content/application_usage/mesh_block_type.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+| .mat        | Matlab triangular ASCII file (read only) |
+| .msh        | GMSH ASCII file                          |
+| .n, .nem    | Sandia's Nemesis format                  |
+| .plt        | Tecplot binary file (write only)         |
+| .node, .ele; .poly | TetGen ASCII file (read; write)   |
+| .inp        | Abaqus .inp format (read only)           |
+| .ucd        | AVS's ASCII UCD format                   |
+| .unv        | I-deas Universal format                  |
+| .xda, .xdr  | libMesh formats                          |
+| .vtk, .pvtu | Visualization Toolkit                    |
```
Comment (sapitts) [inline framework/doc/content/application_usage/mesh_block_type.md]: Could each of these format type explanations under the description heading be updated to include hyperlinks to the associated websites?

Comment (jwpeterson) [reply framework/doc/content/application_usage/mesh_block_type.md]: Good suggestion @sapitts, @leorachapuis is going to add some links for the different formats where available.

Comment (sapitts) [reply framework/doc/content/application_usage/mesh_block_type.md]: Awesome! Thanks @leorachapuis!

**Review by jwpeterson — APPROVED** [review]

**Comment by jwpeterson** [comment]
@brianmoose do you know why these PRs trigger all the MOOSE testing? They only touch .md files and the large_media submodule...

**Comment by brianmoose** [comment]
Probably because large_media. I will see about adding that to the doc check

**Pull request merged**